nautobot/dcim/tests/test_models.py

```diff
@@ -686,49 +695,50 @@ def setUp(self):
 
         DeviceBayTemplate(device_type=self.device_type, name="Device Bay 1").save()
 
-    def test_device_creation(self):
-        """
-        Ensure that all Device components are copied automatically from the DeviceType.
-        """
-        d = Device(
+        self.device = Device(
```

Comment:
Diffs here are noisy but basically just boil down to moving a common `Device` instance out of a bunch of individual test functions and into `setUp` so that it exists for the natural-key tests in `BaseModelTestCase`.